src/agents/customer_group.jl

```diff
@@ -345,47 +357,47 @@ function CustomerGroup(input_filename::AbstractString, model_data::HEMData; id =
         ParamArray(
             "Constant",
             (model_data.index_h,),
-            [0.0, 0.0, 0.0]; 
+            vcat(fill(0.0, 66)),
```

Comment:
This length should not be hard-coded. For zeros, can just use length(model_data.index_h).